tools/misbehaving-jmx-server/configs/benchmark_config.yaml

```
@@ -0,0 +1,13 @@
+domains:
+    test_1:
+        bean_count: 5
+        attribute_count: 2
+        tabular_count: 3
+        composite_count: 4
```

```suggestion
        composite_values_per_tabular_attribute: 4
```

this one is wordy, but something along these lines